chapters/arrays.tex

```diff
@@ -195,10 +195,14 @@ \section{Built-in Array Functions}\doublelabel{built-in-array-functions}
 \lstinline[mathescape=true]!size(C, $j$)! = $1$ for $n_{\mathrm{A}} + 1 \leq j \leq n$, \lstinline[mathescape=true]!C[$i_{1}$, $\ldots$, $i_{n_{\mathrm{A}}}$, 1, $\ldots$, 1]! =
 \lstinline[mathescape=true]!A[$i_{1}$, $\ldots$, $i_{n_{\mathrm{A}}}$]!\\ \hline
 \end{longtable}
+The argument $n$ must be literal or a constant that can be evaluated during translation because it determines
```

Comment:
Since a literal is trivially a constant that can be evaluated during translation, doesn't it suffice to say just _constant_:
```suggestion
The argument $n$ must be a constant that can be evaluated during translation because it determines
```